galoy-client/src/client/mod.rs

```diff
@@ -33,6 +35,7 @@ pub struct GaloyClient {
     client: ReqwestClient,
     config: GaloyClientConfig,
     btc_wallet_id: String,
+    jwt: String,
```

Comment:
Why is `jwt` added here? I don't see the relation with the rest of the PR.

Reply:
See `galoy-client/src/client/mod.rs:179`. The jwt is used in the new client that has the tracing headers injected.